NEFI_functions/fg_assign2.r

```diff
@@ -0,0 +1,89 @@
+#fg_assign.r assigns functional groups to a taxonomy table based on the FUNGuild database.
```

Comment:
Would be great if you could try to document functions using ROxygen syntax/tags rather than just free-form